aldryn_forms/models.py

```diff
@@ -450,6 +450,9 @@ class Option(models.Model):
     def __str__(self):
         return self.value
 
+    class Meta:
+        ordering = ('value', )
```

Comment:
Because `value` can be anything, I think its best to add a dedicated `PositiveIntegerField` which holds the position of the option.